.github/workflows/manual-publish.yml

```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow that is manually triggered
+
+name: Manual Publish Nuget
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+     
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: windows-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        name: Checkout code
+      
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.x'
```

Comment:
We should get this version by a `global.json` file in the repo. I think the action should be able to read the file automatically to gain the version.

Reply:
IMHO, it is not necessary to update the .NET Core version very frequently, .NET Core 3.1 is LTS.

see https://dotnet.microsoft.com/download/dotnet-core